settings.gradle

```diff
@@ -32,3 +32,7 @@ include ':testmodels'
 include ':core-kotlin'
 include ':rxbindings'
 include ':aws-api-appsync'
+
+
+include ':sample-onCall'
```

Comment:
This may have already been discussed, but I didn't see a resolution in the conversation below. Are we going to include a sample project?

Reply:
My bad, will remove it.